pkg/kvstore/kvstore.go

```diff
@@ -35,88 +36,128 @@ const (
 	BaseKeyPrefix = "cilium"
 )
 
-// Get returns value of key
-func Get(key string) ([]byte, error) {
-	v, err := Client().Get(key)
+// ExtendedKVStore is a struct that export multiple kvstore methods
+type ExtendedKVStore struct {
```

Comment:
I'd rather call this just `KVStore`. The names of functions in this PR get too long otherwise.

Reply:
Isn't it a `Client`? Or if you want to keep distinct names, a `KVClient` / `KVStoreClient`?